core/models/country_programme.py

```diff
@@ -8,9 +8,24 @@
     AbstractCPReport,
     AbstractCPUsage,
 )
+from core.models.user import User
```

Comment:
I think the best practice is to use `get_user_model()`: 
https://docs.djangoproject.com/en/5.0/topics/auth/customizing/#django.contrib.auth.get_user_model